.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: usgVZuqrHh2tUUxhEAuhtDRbu3sxkplbR
```

Comment:
This is weird (and bad?) but it looks like CircleCI / Coveralls pairing [requires this](https://github.com/lemurheavy/coveralls-public/issues/1018). Couldn't get anything else to work including adding the token to Circle environment variables.